tests/test_dotenv.py

```diff
@@ -61,3 +61,9 @@ def test_prefix_gets_stripped_once_only(self, mock_envfile):
         settings.load_dotenv_file(mock_envfile)
 
         assert settings.SOME_KEY_PREFIX_REPEATED == "nested"
+
+    def test_as_dict(self, mock_envfile):
+        settings = Settings(env_prefix="PREFIX_")
+        settings.load_dotenv_file(mock_envfile)
+
+        assert settings.as_dict() == settings._config
```

Comment:
Maybe nitpick, but I believe you are not testing that this is a dict (it assumes that `settings._config` is a dict I guess). What do you think about comparing it a literal dict?

Reply:
Yes, make sense. I will add that